.github/workflows/license_check.yml

```diff
@@ -0,0 +1,25 @@
+name: Scancode
+
+on: [pull_request]
+
+jobs:
+  scancode_job:
+    runs-on: ubuntu-latest
+    name: Scan code for licenses
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
I moved to `checkout@v2`.  Seems a bit smarter/faster.

Reply:
ok, will check it out :)